dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -0,0 +1,161 @@
+    "boring-cyborg",
+    "build",
+    "build-providers-dependencies",
+    "chart-schema-lint",
+    "capitalized-breeze",
+    "changelog-duplicates",
+    "check-apache-license",
+    "check-builtin-literals",
+    "check-executables-have-shebangs",
+    "check-extras-order",
+    "check-hooks-apply",
+    "check-integrations",
+    "check-merge-conflict",
+    "check-xml",
+    "daysago-import-check",
+    "debug-statements",
+    "detect-private-key",
+    "doctoc",
+    "dont-use-safe-filter",
+    "end-of-file-fixer",
+    "fix-encoding-pragma",
+    "flake8",
+    "flynt",
+    "codespell",
+    "forbid-tabs",
+    "helm-lint",
+    "identity",
+    "incorrect-use-of-LoggingMixin",
+    "insert-license",
+    "isort",
+    "json-schema",
+    "language-matters",
+    "lint-dockerfile",
+    "lint-openapi",
+    "markdownlint",
+    "mermaid",
+    "mixed-line-ending",
+    "mypy",
+    "mypy-helm",
+    "no-providers-in-core-examples",
+    "no-relative-imports",
+    "pre-commit-descriptions",
+    "pre-commit-hook-names",
+    "pretty-format-json",
+    "provide-create-sessions",
+    "providers-changelogs",
+    "providers-init-file",
+    "providers-subpackages-init-file",
+    "provider-yamls",
+    "pydevd",
+    "pydocstyle",
+    "python-no-log-warn",
+    "pyupgrade",
+    "restrict-start_date",
+    "rst-backticks",
+    "setup-order",
+    "setup-extra-packages",
+    "shellcheck",
+    "sort-in-the-wild",
+    "sort-spelling-wordlist",
+    "stylelint",
+    "trailing-whitespace",
+    "ui-lint",
+    "update-breeze-file",
+    "update-extras",
+    "update-local-yml-file",
+    "update-setup-cfg-file",
+    "update-versions",
+    "verify-db-migrations-documented",
+    "version-sync",
+    "www-lint",
+    "yamllint",
+    "yesqa",
+]
+ALLOWED_INTEGRATIONS = [
+    'cassandra',
+    'kerberos',
+    'mongo',
+    'openldap',
+    'pinot',
+    'rabbitmq',
+    'redis',
+    'statsd',
+    'trino',
+    'all',
+]
+ALLOWED_KUBERNETES_MODES = ['image']
+ALLOWED_KUBERNETES_VERSIONS = ['v1.21.1', 'v1.20.2']
+ALLOWED_KIND_VERSIONS = ['v0.11.1']
+ALLOWED_HELM_VERSIONS = ['v3.6.3']
+ALLOWED_EXECUTORS = ['KubernetesExecutor', 'CeleryExecutor', 'LocalExecutor', 'CeleryKubernetesExecutor']
+ALLOWED_KIND_OPERATIONS = ['start', 'stop', 'restart', 'status', 'deploy', 'test', 'shell', 'k9s']
+ALLOWED_INSTALL_AIRFLOW_VERSIONS = ['2.0.2', '2.0.1', '2.0.0', 'wheel', 'sdist']
+ALLOWED_GENERATE_CONSTRAINTS_MODES = ['source-providers', 'pypi-providers', 'no-providers']
+ALLOWED_POSTGRES_VERSIONS = ['10', '11', '12', '13']
+ALLOWED_MYSQL_VERSIONS = ['5.7', '8']
+ALLOWED_MSSQL_VERSIONS = ['2017-latest', '2019-latest']
+ALLOWED_TEST_TYPES = [
+    'All',
+    'Always',
+    'Core',
+    'Providers',
+    'API',
+    'CLI',
+    'Integration',
+    'Other',
+    'WWW',
+    'Postgres',
+    'MySQL',
+    'Helm',
+    'Quarantined',
+]
+ALLOWED_PACKAGE_FORMATS = ['both', 'sdist', 'wheel']
+ALLOWED_USE_AIRFLOW_VERSION = ['.', 'apache-airflow']
+
```

Comment:
I have added ALLOWED values in the constants file. Can you check and share your views? @potiuk

Reply:
Looks good!